webapp/src/css/enketo/medic.less

```diff
@@ -384,6 +384,10 @@
   .question input[type=checkbox]:checked {
     background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' stroke='%234f93ce' fill='%234f93ce' width='32' height='32' viewBox='0 0 32 32'%3E%3Cpath d='M25.1 12.5l-3.4-3.3-8 8-2.9-3-3.4 3.4 6.3 6.3z'/%3E%3C/svg%3E");
   }
+  
+  .pages.or .or-repeat-info[role="page"] {
+    display: block !important;
```

Comment:
Is the `!important` required?

Reply:
It's recommended to never use `!important` in CSS because it means that if another stylesheet wants to override it in some specific case then it also needs to use `!important`. Eventually you end up just using `!important` everywhere and then it's pointless. Instead you can override styles with more specificity which makes the CSS easier to maintain into the future.

Read this post for more info: https://j11y.io/css/dont-use-important/

Reply:
Point noted about using `!important` and I've removed it.